cmd/imgcollector/image/image.go

```diff
@@ -34,3 +36,10 @@ func LayerIDs(img v1.Image) ([]string, error) {
 	}
 	return layerIDs, nil
 }
+
+func NamespacedRegistry(ref name.Reference) string {
+	fullName := ref.Context().Name()
+	endOfRegistry := strings.Index(fullName, "/")
+	endOfNamespace := strings.LastIndex(fullName[endOfRegistry:], "/")
+	return fullName[:endOfNamespace-1] // [:endOfNamespace)
```

Comment:
What is this comment `[:endOfNamespace)`?

Reply:
https://en.wikipedia.org/wiki/Interval_(mathematics)